README.md

```diff
@@ -20,6 +20,7 @@ This repository is intended to be a base template, a cookiecutter for a new Pyth
 [Git hooks](#git-hooks)  
 [Testing](#testing)  
 [Generating documentation](#generating-documentation)  
+[Synchronizing with this template repo](#Synchronizing-with-this-template-repo)  
```

Comment:
I think (not sure without checking) that the generated heading id is all lower-case: `#synchronizing-with-this-template-repo`

Reply:
Commit [4108c96](https://github.com/jenstroeger/python-package-template/pull/444/commits/4108c96fdffcb4ac39a0bba5abaaaa3ca45c6b27).